pcdsdevices/slits.py

```diff
@@ -492,3 +499,55 @@ class PowerSlits(BeckhoffSlits):
 
     rtds = DDCpt(_rtd_fields(RTD, 'rtd', range(1, 9)))
     fsw = Cpt(NotImplementedSignal, ':FSW', kind='normal')
+
+
+class ExitSlits(BaseInterface, Device, LightpathInOutMixin):
+    tab_component_names = True
+
+    lightpath_cpts = ['target']
+    _icon = 'fa.video-camera'
```

Comment:
I think the only thing I would have suggested was a different lightpath icon

Reply:
Not that we're using lightpath or anything

Reply:
That makes me sad....

Reply:
@ZLLentz The target is the Yag on the exit slits, and I thought that is what we use. I can change it. what icons should it be?

Reply:
I'm not sure, but I'm pretty sure that if someone is looking for "slits" they will glance over a video camera icon. I couldn't find a better option when I post-reviewed this (something to capture both elements- slits and camera) so I don't really have much standing here.

Reply:
I wasn't thinking of the slits. I was thinking of only the Yag in the beam path. But if we need to include the slits, then can we add two icons?

Reply:
That's a good point, only the yag blocks beam, the slits are more an energy selector than a beam shaper.  We should leave it for now.